libs/rbcapi/src/bills/index.test.ts

```diff
@@ -0,0 +1,36 @@
+
+import  { getEnvVars } from "../../../../tools/setenv";
+import { describe, IsManualRun } from '@thecointech/jestutils';
+import { prepareBillPayee } from '.';
+import * as manage from './managePayees';
+import { ApiAction, initBrowser } from '../action';
+
+const vars = getEnvVars("prod");
```

Comment:
the problem with scraping.  Our tests aren't really testing our code, they are more for testing whether or not that code that worked last week still works.

As such, they kinda need production vars to run (this means they don't run on CI)